### ISSUE TITLE
GitHub user and organization accounts look identical in fast user switching UI

### ISSUE THREAD

**Issue by timtailscale** [tailscale/tailscale#9286]
### What is the issue?

If you login to Tailscale using GitHub, and authenticate your device with both a individual GitHub account, and an organization that your individual account is a part of the fast user switching UI will look identical so you cannot tell which is the org and which is the individual account. This applies to both macOS and Windows.

![image](https://github.com/tailscale/tailscale/assets/134532514/804c7f72-4977-4308-b60a-efa40ec2c9a0)

### Steps to reproduce

Authenticate a single device with a GitHub tailnet, and GitHub org tailnet using the same GitHub account.

### OS

macOS, Windows

### Tailscale version

1.48.1

**Comment by soniaappasamy** [comment]
I think this may be an issue with orgs displaying wrong specifically in the case where you have also logged into the personal github tailnet associated with that account. In my local setup, I don't see the org for soniaappasamy (one of the two is my personal, the other is an org), but I do see the org displaying correctly for two test accounts that logged into an org where I have _never_ logged into their own tailnets. Just a suspicion

<img width="289" alt="Screenshot 2023-09-07 at 4 37 28 PM" src="https://github.com/tailscale/tailscale/assets/9019214/673f794c-6bca-4eb6-bada-f03eb360046e">

**Comment by DenWav** [comment]
This didn't use to be an issue, it used to be the org account would have a specifier in parentheses, but that's not true anymore.

This issue also exists on the CLI:
```
$ tailscale switch --list
DenWav@github *
DenWav@github
```

**Comment by freekvh** [comment]
I am experiencing the same issue since today (I update my nixos channel yesterday), i can't switch tailnets anymore because they are identical:

```bash
[freek@nixos:~]$ sudo tailscale switch --list
[sudo] password for freek: 
freekvh@github
freekvh@github *
```

**Comment by freddyheppell** [comment]
GitHub organisation names are now showing up for me again in the macOS app. I'm unsure which version specifically, and I can't see anything in the release notes about it.

**Comment by soniaappasamy** [comment]
Tailnet names started getting filled in the 1.56 release https://github.com/tailscale/tailscale/releases/tag/v1.56.0

**Comment by marwan-at-work** [comment]
@freekvh 👋🏼 given the output format, I believe it's an older version so updating to 1.56+ should resolve it

Your output format should look like this after you upgrade: 

```
$ tailscale switch --list
ID    Tailnet                  Account
17de  marwan-at-work.github    marwan-at-work@github
fe6c  examples.org.github      marwan-at-work@github
a673  otheraccounts@gmail.com  marwan.sameer@gmail.com*
```

A minor inconvenience to note:  once you have v1.56+ installed, you will have to switch between the tailnets in your GUI/CLI in order for the new data to be refreshed and those duplications to get resolved. 

In other words, if you see the `Tailnet` column in the above output to be empty for each row, then doing `tailnet switch <id>` should refresh the data and the next time you `tailscale switch --list` should be fixed forever.

**Comment by marwan-at-work** [comment]
@freddyheppell 👋🏼  likewise on my account above, but just with the mac GUI instead of the CLI. If you still see this issue even after 1.56 (and switching between the tailnets to update the data) please feel free to comment here. Thanks!

**Comment by freekvh** [comment]
Yes, fixed for me:

```
$ sudo tailscale switch --list
[sudo] password for freek: 
ID    Tailnet             Account
225b  my-org.github  freekvh@github*
26e4                      freekvh@github
```

```
$ tailscale version
1.56.1
  go version: go1.21.4
```

**Comment by DentonGentry** [comment]
I believe this is resolved in the 1.56 release.
